GPU/GLES/Framebuffer.cpp

```diff
@@ -1348,7 +1359,7 @@ void FramebufferManager::BlitFramebuffer(VirtualFramebuffer *dst, int dstX, int
 		// Should maybe revamp that interface.
 		float srcW = src->bufferWidth;
 		float srcH = src->bufferHeight;
-		DrawActiveTexture(0, dstX1, dstY, w * dstXFactor, h, dst->bufferWidth, dst->bufferHeight, !flip, srcX1 / srcW, srcY / srcH, srcX2 / srcW, (srcY + h) / srcH, draw2dprogram_);
+		DrawActiveTexture(0, dstX1, dstY, w * dstXFactor, h, dst->bufferWidth, dst->bufferHeight, srcX1 / srcW, srcY / srcH, srcX2 / srcW, srcY2 / srcH, draw2dprogram_);
```

Comment:
Hmm, this is a bit confusing, since the factors are fixed to 1 here.  Should probably change `dstY` to `dstY1` and `srcY` to `srcY1` to match, and either add `* dstYFactor` to `h` or remove from `w`.  Before y was always "unscaled" in this statement, although that's sorta confusing anyway.

-[Unknown]